eng/packages/http-client-csharp/generator/TestProjects/Local/Mgmt-TypeSpec/src/Generated/Models/ExtendedLocationType.cs

```diff
@@ -9,7 +9,7 @@
 using System.ComponentModel;
 using MgmtTypeSpec;
 
-namespace MgmtTypeSpec.Models
+namespace Azure.ResourceManager.CommonTypes.Models
```

Comment:
should this be changed back to `MgmtTypeSpec.Models`?

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/48197#discussion_r1948699268

Reply:
OK, there could be models not replaceable in Azure.ResourceManager. We need to update the namespace for Azure.

Reply:
updated